stdlib/REPL/test/replcompletions.jl

```diff
@@ -2484,3 +2484,9 @@ let (c, r, res) = test_complete_context("global xxx::Number = Base.", Main)
     @test res
     @test "pi" ∈ c
 end
+
+# JuliaLang/julia#57780
+const issue57780 = ["a", "b", "c"]
+const issue57780_orig = copy(issue57780)
+test_complete_context("empty!(issue57780).", Main)
```

Comment:
Imo it would be good to have a bit lower level test as well. Some refactoring in the REPL code might cause the original issue to no the hit.

Reply:
yeah, we should add it.